react/components/FilterNavigator/legacy/SideBar.js

```diff
@@ -58,7 +70,7 @@ class Sidebar extends Component {
 
 Sidebar.propTypes = {
   /* Internationalization */
-  intl: intlShape.isRequired,
+  intl: PropTypes.any,
```

Comment:
Why changed from intlShape.isRequired to any?

Reply:
Because the latest version of react-intl doesn't provide the intlShape to use as prop types